README.md

```diff
@@ -26,6 +26,8 @@ In bigger projects with many files it also provides **context**, it gives you a
 
 - Log output of all git commands run
 
+- Search within changed files (Git Tree Comparse: Search Changes)

```

Comment:
```suggestion
- Search within changed files

```

Let's keep this list simple, features should be discoverable easily via the UI.

Reply:
Done!